tests/test.rs

```diff
@@ -18,7 +29,7 @@ fn create_socketpair() -> (Socket, Socket) {
     receiver.set_rcvtimeo(1000).unwrap();
 
     receiver.bind("tcp://*:*").unwrap();
```

Comment:
Fundamentally, the issue is on windows one cannot connect() to the any address "0.0.0.0", as stated [here](https://msdn.microsoft.com/en-us/library/windows/desktop/ms737625.aspx):
````
If the address member of the structure specified by the name parameter is filled with zeros, connect will return the error WSAEADDRNOTAVAIL.
````
This is the address that is returned from get_last_endpoint() after binding the wildcard address ``"tcp://*:*"``.

If you'd like to avoid this in the test, then simply switching the bind to "127.0.0.1:*" works, and is arguably an equally effective test.

I don't think there is an effective way to bubble the "fix" into the library itself (i.e. into get_last_endpoint()); doing the ``replace`` fix done in this patch inside the actual library seems like a potentially unpleasant unexpected/hidden behavior...